### DISCUSSION TITLE
5.0.6.Final

### DISCUSSION THREAD

**Discussion by jamezp** [resteasy/resteasy#3504] [Announcements]
## What's Changed
* [RESTEAST-3274] Upgrade mime4j to 0.8.9. by @jamezp in https://github.com/resteasy/resteasy/pull/3383
* [RESTEASY-3280] Do not load the ThreadContext's from a service loader… by @jamezp in https://github.com/resteasy/resteasy/pull/3396
* [RESTEASY-3283] Do not throw a DefaultOptionsMethodException when no OPTIONS method is found. Instead, just return the created response. by @jamezp in https://github.com/resteasy/resteasy/pull/3412
* [RESTEASY-3286] Use the new file utilities to create temporary files. by @jamezp in https://github.com/resteasy/resteasy/pull/3412
* Revert "[RESTEASY-2845] ResteasyWebTarget.proxy(Class) problem with questionmark in @Path with Regex " by @jamezp in https://github.com/resteasy/resteasy/pull/3433
* [RESTEASY-3290] Avoid an NPE if the temporary directory is null and e… by @jamezp in https://github.com/resteasy/resteasy/pull/3437
* Bump actions/download-artifact from 2 to 3 by @dependabot in https://github.com/resteasy/resteasy/pull/3443
* Bump actions/setup-java from 2 to 3 by @dependabot in https://github.com/resteasy/resteasy/pull/3444
* Bump actions/upload-artifact from 2 to 3 by @dependabot in https://github.com/resteasy/resteasy/pull/3446
* Bump actions/checkout from 2 to 3 by @dependabot in https://github.com/resteasy/resteasy/pull/3445
* Component upgrades 5.0 by @jamezp in https://github.com/resteasy/resteasy/pull/3470
* Bump arquillian-jetty-embedded-9 from 1.0.0.CR3 to 1.0.0.Final by @dependabot in https://github.com/resteasy/resteasy/pull/3447
* Bump FastInfoset from 1.2.17 to 1.2.18 by @dependabot in https://github.com/resteasy/resteasy/pull/3448
* Bump version.io.netty.netty4 from 4.1.78.Final to 4.1.89.Final by @dependabot in https://github.com/resteasy/resteasy/pull/3455
* Bump version.com.fasterxml.jackson from 2.12.6 to 2.12.7 by @dependabot in https://github.com/resteasy/resteasy/pull/3461
* Bump reactive-streams from 1.0.3 to 1.0.4 by @dependabot in https://github.com/resteasy/resteasy/pull/3463
* Bump version.org.eclipse.jetty from 9.4.39.v20210325 to 9.4.51.v20230217 by @dependabot in https://github.com/resteasy/resteasy/pull/3468
* Bump version.org.jboss.marshalling.jboss-marshalling from 2.0.6.Final to 2.0.13.Final by @dependabot in https://github.com/resteasy/resteasy/pull/3469
* Bump galleon-maven-plugin from 4.2.8.Final to 4.2.9.Final by @dependabot in https://github.com/resteasy/resteasy/pull/3472
* Component upgrades 5.0 by @jamezp in https://github.com/resteasy/resteasy/pull/3485
* Bump jakarta.inject-api from 1.0 to 1.0.5 by @dependabot in https://github.com/resteasy/resteasy/pull/3475
* Bump httpcore from 4.4.14 to 4.4.16 by @dependabot in https://github.com/resteasy/resteasy/pull/3476
* Bump mockito-core from 3.4.0 to 3.4.6 by @dependabot in https://github.com/resteasy/resteasy/pull/3477
* Bump jboss-marshalling-osgi from 2.0.5.Final to 2.0.13.Final by @dependabot in https://github.com/resteasy/resteasy/pull/3478
* Bump jboss-logging from 3.4.2.Final to 3.4.3.Final by @dependabot in https://github.com/resteasy/resteasy/pull/3482
* Bump arquillian-weld-embedded from 2.0.0.Final to 2.0.1.Final by @dependabot in https://github.com/resteasy/resteasy/pull/3480
* Component upgrades 5.0 by @jamezp in https://github.com/resteasy/resteasy/pull/3493
* Bump version.io.undertow from 2.2.18.Final to 2.2.23.Final by @dependabot in https://github.com/resteasy/resteasy/pull/3488
* Bump httpclient from 4.5.13 to 4.5.14 by @dependabot in https://github.com/resteasy/resteasy/pull/3490
* Bump jboss-threads from 2.3.3.Final to 2.3.6.Final by @dependabot in https://github.com/resteasy/resteasy/pull/3491
* Bump yasson from 1.0.9 to 1.0.11 by @dependabot in https://github.com/resteasy/resteasy/pull/3492
* Component upgrades 5.0 by @jamezp in https://github.com/resteasy/resteasy/pull/3499
* Bump httpasyncclient from 4.1.4 to 4.1.5 by @dependabot in https://github.com/resteasy/resteasy/pull/3494
* Bump jboss-jaxrs-api_2.1_spec from 2.0.1.Final to 2.0.2.Final by @dependabot in https://github.com/resteasy/resteasy/pull/3495
* Bump wildfly-galleon-maven-plugin from 5.2.2.Final to 5.2.11.Final by @dependabot in https://github.com/resteasy/resteasy/pull/3496
* Bump jakarta.mail from 1.6.5 to 1.6.7 by @dependabot in https://github.com/resteasy/resteasy/pull/3497
* Bump arquillian-bom from 1.7.0.Alpha10 to 1.7.0.Alpha14 by @dependabot in https://github.com/resteasy/resteasy/pull/3498

**Full Changelog**: https://github.com/resteasy/resteasy/compare/5.0.5.Final...5.0.6.Final

<hr /><em>This discussion was created from the release <a href='https://github.com/resteasy/resteasy/releases/tag/5.0.6.Final'>5.0.6.Final</a>.</em>

**Comment by dovezp** [comment]
@jamezp please update https://github.com/resteasy/resteasy/releases/tag/5.0.6.Final with disclosure of https://github.com/resteasy/resteasy/commit/4feb572249c292bb4861027674502c6b851fb132 instead of "[5.0] Back ported fixes by @jamezp in https://github.com/resteasy/resteasy/pull/3412"

**Reply by jamezp** [reply]
Sure. Sorry, I just use the generated change log. I'll update it now.

**Reply by dovezp** [reply]
tyvm it was throwing off the CVE reports.
